doc/dev/installation.rst

```diff
@@ -350,7 +350,7 @@ them, but using the ``test-runtime`` target instead:
 
 .. code-block:: console
 
-  make test-runtime ENABLE_LIGHTNING_KOKKOS=ON ENABLE_OPENQASM=ON
```

Comment:
We might want to think about what happens with the runtime tests Ali wrote for the device. Were they migrated over? Or are we just dropping them? @maliasadi

Reply:
Yes, they are all migrated to the Lightning repo and we can drop them all in the runtime.